src/components/side-panel/add-elements-panel/add-elements-panel.tsx

```diff
@@ -51,11 +92,45 @@ export const AddElementsPanel = () => {
                         {renderElementButtons(category.elementTypes)}
                     </ElementButtonList>
                 </Category>
+            ));
+        } else {
+            return (
+                <NoResultsWrapper>
+                    <NotFoundIcon />
+                    <Text.Body>No results found.</Text.Body>
+                </NoResultsWrapper>
             );
-        });
+        }
     };
 
     return (
-        <Wrapper data-testid="add-elements-panel">{renderCategories()}</Wrapper>
+        <Wrapper data-testid="add-elements-panel">
+            <label>
+                <ElementSearch
+                    placeholder="Search"
+                    addon={{
+                        type: "custom",
+                        attributes: {
+                            children: (
+                                <MagnifierIcon
+                                    onClick={handleMagnifierClick}
```

Comment:
no need onClick anymore, remember to remove `handleMagnifierClick` function

Reply:
has been updated in the latest changes